src/components/atoms/FilmDescriptionList/FilmDescriptionList.tsx

```diff
@@ -1,33 +1,33 @@
 import * as React from "react";
 import { Ifilm } from "../../../types";
-
+import "./index.css";
 interface IFilmDescriptionList {
   film: Ifilm;
 }
 
-export const FilmDescriptionList = (film: Ifilm) => {
+export const FilmDescriptionList = ({ film }: IFilmDescriptionList) => {
```

Comment:
({ film }: IFilmDescriptionList)  -> (film: Ifilm) 

and Ifilm -> IFilm (rename please)